lib/ansible/modules/cloud/vmware/vmware_host_firewall_manager.py

```diff
@@ -138,6 +185,12 @@ def gather_rule_set(self):
                 for rule_set_obj in firewall_system.firewallInfo.ruleset:
                     temp_rule_dict = dict()
                     temp_rule_dict['enabled'] = rule_set_obj.enabled
+                    allowed_host = rule_set_obj.allowedHosts
+                    rule_allow_host = dict()
+                    rule_allow_host['ip_address'] = [ip for ip in allowed_host.ipAddress]
```

Comment:
I'm not sure why you loop on allowed_host.ipAddress if it's already a list.

Reply:
Changed to remove list logic and just assign the property.